torch/csrc/distributed/c10d/comm.h

```diff
@@ -13,4 +14,63 @@ void broadcast_coalesced(
     at::TensorList tensors,
     size_t buffer_size);
 
+// This class passes bucket contents tensor to DDP communication hook.
+// Optionally in the future this can be enhanced with parameter to bucket
+// mappings as well.
+class GradBucket {
```

Comment:
in the comment, it can clarify the tensors are bucket content tensors for multiple replicas